browser/components/newtab/RemoteAboutNewTab.jsm

```diff
@@ -424,10 +434,10 @@ let RemoteAboutNewTab = {
    * also clear the links cache and update the set of links to display, as they
    * may have changed, then proceed with the page update.
    */
-  observe: function(aSubject, aTopic, aData) {
+  observe: function(aSubject, aTopic, aData) { // jshint ignore:line
```

Comment:
Nit: We've been kinda ignoring Hungarian notation in this JSM (i.e., for all the `message` args)... I wonder if we should do the same here?